src/Tokenizer.ts

```diff
@@ -284,16 +284,18 @@ export class _Tokenizer {
         if (this.options.pedantic) {
           indent = 2;
           itemContents = line.trimStart();
+        } else if (!line.trim()) {
+          indent = cap[1].length + 1;
         } else {
           indent = cap[2].search(/[^ ]/); // Find first non-space char
           indent = indent > 4 ? 1 : indent; // Treat indented code blocks (> 4 spaces) as having only 1 indent
           itemContents = line.slice(indent);
           indent += cap[1].length;
         }
 
-        let blankLine = false;
+        let blankLine = !line.trim();
 
-        if (!line && /^ *$/.test(nextLine)) { // Items begin with at most one blank line
+        if (!line.trim() && /^ *$/.test(nextLine)) { // Items begin with at most one blank line
```

Comment:
```suggestion
        if (blankLine && /^ *$/.test(nextLine)) { // Items begin with at most one blank line
```
Can you reuse the variable?

I'm also curious if you can hoist line 296 up to line 281 so you can use the variable again on line 287.